src/foam/nanos/ruler/UserRefines.js

```diff
@@ -9,7 +9,7 @@ foam.CLASS({
     name: 'UserRefines',
     refines: 'foam.nanos.auth.User',
     imports: [
-      'complianceHistoryDAO'
+      'complianceHistoryDAO?'
```

Comment:
We can't have this on the foam side. This needs to be a nanopay userRefinement.

Reply:
removed this from foam and placed in nanopay.model.User